tests/infra/ccf.py

```diff
@@ -278,7 +278,7 @@ def _add_node(self, node, lib_name, args, target_node=None, should_wait=True):
             node.network_state = NodeNetworkState.joined
 
     def _wait_for_node_to_exist_in_store(
-        self, remote_node, node_id, node_status=None, timeout=3
+        self, remote_node, node_id, node_status=None, timeout=5
```

Comment:
Is the timeout change necessary?

Reply:
Yes it seems like it, the fourth node sometimes has trouble joining, especially when there are nodes lagging behind and the primary is busy catching them up. In any case trying for 5 seconds instead of 3 won't hurt since if the node starts sooner then this will terminate early anyway.